rebasehelper/helpers/bugzilla_helper.py

```diff
@@ -0,0 +1,152 @@
+    BUGZILLA_REST = 'https://bugzilla.redhat.com/rest/bug'
+
+    @classmethod
+    def get_bugzilla_component(cls, bugzilla_id: str) -> str:
+        """Gets a component of the bugzilla.
+
+        Args:
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Component of the bugzilla.
+
+        Raises:
+            RebaseHelperError: If no such bugzilla exists or if the bugzilla
+            is not created by Upstream Release Monitoring.
+
+        """
+        r = requests.get('{}/{}'.format(cls.BUGZILLA_REST, bugzilla_id))
+        if not r.ok:
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+
+        if 'error' in r.json():
+            raise RebaseHelperError('No bugzilla with ID {} was found'.format(bugzilla_id))
+
+        bug = r.json()['bugs'][0]
+        if bug['creator_detail']['email'] != 'upstream-release-monitoring':
+            raise RebaseHelperError('The given bugzilla was not created by Upstream Release Monitoring')
+        return bug['component'][0]
+
+    @classmethod
+    def get_version_from_comments(cls, bugzilla_id: str) -> Optional[str]:
+        """Gets version from bugzilla comments.
+
+        Args:
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Version specified by Upstream Release Monitoring in comments
+            or None, if no version could be found.
+
+        Raises:
+            RebaseHelperError: If no such bugzilla exists.
+
+        """
+        r = requests.get('{}/{}/comment'.format(cls.BUGZILLA_REST, bugzilla_id))
+        if not r.ok:
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+        version = None
+        comments = r.json()['bugs'][bugzilla_id]['comments']
+        pattern = re.compile(r'^Latest upstream release: (?P<version>.*)\n')
+        for comment in comments:
+            if comment['creator'] != 'upstream-release-monitoring':
+                continue
+            match = pattern.match(comment['text'])
+            if match:
+                version = match.group('version')
+
+        return version
+
+    @classmethod
+    def clone_repository(cls, component: str, bugzilla_id: str) -> str:
+        """Clones the component package repository
+
+        Args:
+            component: Package to clone.
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Path to the cloned repository.
+
+        Raises:
+            RebaseHelperError: If the directory, that the repository
+            is supposed to be cloned into, exists.
+
+        """
+        path = os.path.abspath('{}-{}'.format(bugzilla_id, component))
+        if os.path.exists(path):
+            raise RebaseHelperError('Could not clone the repository because the directory '
+                                    '{} already exists'.format(path))
+
+        url = cls.REPO_URL.format(component)
+        logger.info("Cloning %s into %s", url, path)
```

Comment:
Verbose output now looks like this:

```
INFO: Cloning https://src.fedoraproject.org/rpms/gpxsee.git into /tmp/1756742-gpxsee
VERBOSE: Rebase-helper version: 0.18.1.dev33+g3b749cd
VERBOSE: argument passed as a new source is a version
```

I think rebase-helper version should always be the very first message.
Also the third message is obviously wrong, but that's unrelated to this PR.